app/views/sessions/new.html.erb

```diff
@@ -0,0 +1,23 @@
+
+<%= form_with model: @user, url: '/login', method: :post do |form| %>
+  <% if notice %>
```

Comment:
extract this to a partial and use it everywhere

Reply:
Done